src/ansys/sherlock/core/types/analysis_types.py

```diff
@@ -15,67 +14,103 @@ class ElementOrder:
     """Constants for Element Order."""
 
     LINEAR = analysis_service.ElementOrder.Linear
+    "LINEAR"
```

Comment:
Documentation isn't generated in the html when these members don't have a comment.